applications/asset_tracker/Kconfig

```diff
@@ -323,6 +323,29 @@ endif
 
 endmenu # Environment sensors
 
+menu "Light Sensor"
+
+config BH1749
```

Comment:
I'm no Kconfig expert, but I think this should rather be structured something like
```
menuconfig LIGHT_SENSOR
    bool "Light sensor"
    depends on BH1749
    default y if NRF9160_PCA20835NS

if LIGHT_SENSOR
... 
```
instead of configuring symbols in a driver directly.

I'd also like a `config LIGHT_SENSOR_DEV_NAME` that would let us add light sensor capability to the sensor simulator to be used for PCA10090, or hook up any other light sensor to the DK and use that.

Reply:
updated, please review.
i'm certainly not a kconfig expert 🙃